indico/modules/events/abstracts/templates/management/notification_tpl_form.html

```diff
@@ -1,10 +1,9 @@
 {% from 'forms/_form.html' import form_header, form_footer, form_rows, form_row, form_fieldset %}
 
 {{ form_header(form) }}
-    {{ form_row(form.title) }}
+    {{ form_rows(form, fields=('title', 'stop_on_match')) }}
```

Comment:
Why do we need the field "Default template" to be at the bottom and not together with the "Title" and "Stop on match"?